components/dropdown/dropdown-content-mixin.js

```diff
@@ -692,9 +779,9 @@ export const DropdownContentMixin = superclass => class extends RtlMixin(supercl
 			'd2l-dropdown-content-footer': this._hasFooter
 		};
 
-		return html`
+		const dropdown =  html`
 			<div class="d2l-dropdown-content-position" style=${styleMap(positionStyle)}>
-				<div class="d2l-dropdown-content-width" style=${styleMap(widthStyle)}>
+				<div  id="d2l-dropdown-wrapper" class="d2l-dropdown-content-width" style=${styleMap(widthStyle)} ?closing="${this._closing}">
```

Comment:
Technically we shouldn't be putting non-standard attributes like `closing` on native elements like a `div`. I'd recommend using something like `?data-closing="${this._closing}"`.